src/models/verticalConfig.ts

```diff
@@ -7,11 +7,11 @@ import { VerticalLink } from './verticalLink';
  *
  * @public
  */
-export interface VerticalConfig {
+export interface VerticalConfig<T = Record<string, unknown>> {
```

Comment:
Do we want `Record<string, any>` as the default instead? By having `unknown`, will it put a lot of burden on the user? Specifically, I'm wondering if they'll be forced to do manual, run-time checks in their cards. cc: @yen-tt

Reply:
After talking with Yen and Davish, we do want `unknown`.